README.md

```diff
@@ -17,8 +17,8 @@ configType: java
 configVersion: 1
 # REQUIRED - The main class to be run
 mainClass: my.package.Main
-# OPTIONAL - Path to the JRE, defaults to the JAVA_HOME environment variable if unset
-javaHome: javaHome
+# OPTIONAL - Path to the JRE or environment variable name (e.g. $JAVA_11_HOME). Defaults to the JAVA_HOME environment variable if unset
```

Comment:
$JAVA_11_HOME or "JAVA_11_HOME"? We need to make sure that environment variables passed here are not evaluated (ie we need to disallow `$(rm -rf /)`)

Reply:
Yep don't worry, the dollar will only ever appear in a `launcher-static.yml` on disk and then immediately expanded by go, it never gets put in a shell!

Reply:
reading the Go code, it seems like you might eval the env var though? Could we have a test to check that `$(<stuff>)` definitely doesn't execute?